packages/distributor/.eslintrc

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
you may place one root eslint config in the root directory and refer them in each package via `extends` to this root config. I guess each package has duplicated config and it's going to be great to have similar formatting across the whole repo